ios/FluentUI.Demo/FluentUIDemoTests/ActivityIndicatorTest.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import XCTest
+
+class ActivityIndicatorTest: BaseTest {
+    override var controlName: String { "ActivityIndicator" }
+
+    override func setUpWithError() throws {
+        try super.setUpWithError()
+        continueAfterFailure = false
+    }
```

Comment:
Hmm do we still need to override setupwitherror for this test? Makes sense to override it for tests like ActivityIndicatorTestSwiftUI where there's additional set up, but here it seems like the only thing we need it for is to set continueAfterFailure (which it seems like we already did in the base test)?